operator/README.md

```diff
@@ -0,0 +1,175 @@
+<!-- TOC -->
+- [PMEM-CSI Operator](#pmem-csi-for-kubernetes)
+  - [About](#about)
+  - [Usage](#usage)
+    - [Pre-requisites](#pre-requisites)
+    - [Deploy PMEM-CSI operator](#deploy-pmem-csi-operator)
+    - [Deployment CRD API](#deployment-crd-api)
+    - [Driver certificates](#driver-certificates)
+  - [Example Deployment](#example-deployment)
+
+<!-- /TOC -->
+# PMEM-CSI operator
+
+## About
+
+PMEM-CSI operator facilitates deploying and managing the [PMEM-CSI driver](https://github.com/intel/pmem-csi) on a Kubernetes cluster. 
+
+This operator is based on the CoreOS [operator-sdk](https://github.com/operator-framework/operator-sdk) tools and APIs.
+
+## Usage
+
+### Pre-requisites
+
+A running Kubernetes cluster with one or more nodes installed with PMEM hardware(NVDIMM). The persistent memory on these nodes must be [pre-provisioned](../README.md#persistent-memory-pre-provisioning).
+
+### Deploy PMEM-CSI operator
+
+<!-- The assumptions:
+  a) pmem-csi-operator binary is part of released intel/pmem-csi-driver image and no additional steps required to build the images
+-->
+
+```sh
+$ kubectl create -f https://raw.githubusercontent.com/intel/pmem-csi/operator/operator/deploy/operator.yaml
```

Comment:
Better use the URL before the redirect, then it is simpler to get the non-raw version by stripping out "/raw/":

```
https://github.com/intel/pmem-csi/raw/operator/operator/deploy/operator.yaml
```

Please also add a TODO about replacing the first "operator" with "devel" after merging into "devel" and a second TODO about patching this URL in release branches.

Reply:
Done.